packages/common/src/ide/normalized/NormalizedIDE.ts

```diff
@@ -46,7 +46,7 @@ export class NormalizedIDE extends PassthroughIDEBase {
       hatStability: this.configuration.getOwnConfiguration(
         "experimental.hatStability",
       ),
-      snippetsDir: getFixturePath("cursorless-snippets"),
+      snippetsDir: this.cursorlessSnippetsDir,
```

Comment:
This implementation I'm not quite happy with. The problem is that normalized ide is used at runtime and uses a environment fixture path. I will happily take any suggestions here.

Reply:
I mean `NormalizedIDE` is really test / dev code, so probalby shouldn't be in engine / common. But I'm ok with this for now